packages/core/src/physics/joint/HingeJoint.ts

```
@@ -105,24 +98,27 @@
-      this._jointMonitor = value;
-      (<IHingeJoint>this._nativeJoint).setDriveVelocity(value.targetVelocity);
-      (<IHingeJoint>this._nativeJoint).setDriveForceLimit(value.forceLimit);
-      (<IHingeJoint>this._nativeJoint).setDriveGearRatio(value.gearRation);
-      (<IHingeJoint>this._nativeJoint).setHingeJointFlag(HingeJointFlag.DriveFreeSpin, value.freeSpin);
+    if (this._jointMotor !== value) {
+      if (this._jointMotor) {
+        this._jointMotor._updateFlagManager.removeListener(this._onMotorChanged);
+      }
+
+      this._jointMotor = value;
+      value && value._updateFlagManager.addListener(this._onMotorChanged);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)